packages/utils/.eslintignore

```diff
@@ -0,0 +1,13 @@
+cjs/
```

Comment:
if we don't, we should also publish as many eslint configs as possible to our own respond-esling

 (I think james is wanting to change the name from respond-framework to respond - which i agree with so lemme know if thats a mission and ill backlog it

Reply:
prioritization here, this is looooow, dont waste your time on this - ill find more junior or do it myself <3

Reply:
actually, someone else owns the npm name `respond` I've asked if he'll hand over his 6-year-old name.

Reply:
sure, whatever works. It should be a simple find/replace to change `@respond-framework` to `@respond` or `@rudy`. Shorter is better :+1:

Reply:
hmm that's a good idea to publish the eslint config and such things - maybe we can have `@respond-framework/config` or something to keep babel/eslint configs and the like - might be a bit cleaner than putting them all in the root package.

https://github.com/respond-framework/rudy/pull/9 will probably make that easier - at least after that there's alot less duplication.

Reply:
created https://github.com/respond-framework/rudy/issues/10 to track this